src/components/controls/AlertNotice.svelte

```diff
@@ -0,0 +1,73 @@
+<script>
+  import { fade } from 'svelte/transition';
+  import { cubicOut } from 'svelte/easing';
+
+  // So we don't pollute the entire LS namespace.
+  const KEY_PREFIX = 'alertNotice-';
+  export let dismissText = 'dismiss'; // Label of 'dismiss' button.
+
+  // Required: Which localStorage key to check whether this notice has been dismissed.
+  export let toggleKey = '';
+
+  $: alertVisible = window.localStorage.getItem(KEY_PREFIX + toggleKey) !== 'true';
+
+  function dismissNotice() {
+    alertVisible = false;
+    window.localStorage.setItem(KEY_PREFIX + toggleKey, 'true');
+  }
+</script>
+
+<style>
+  .alert-notice {
+    position: absolute;
+    left: var(--space-4x);
+    bottom: var(--space-4x);
+    border: 1px solid var(--bright-yellow-500);
+    display: flex;
+    background-color: var(--bright-yellow-100);
+    color: var(--bright-yellow-700);
+    border-radius: var(--border-radius-base);
+    padding: var(--space-2x);
+    align-items: center;
+    grid-gap: var(--space-3x);
+    width: fit-content;
+  }
+
+  .alert-notice-content {
+    line-height: 1.5;
+    font-size: var(--text-015);
+    width: 350px;
+  }
+
+  .alert-notice-content > * {
+    margin: 0;
+  }
+
+  .alert-notice-action button {
+    padding: var(--space-base);
```

Comment:
![jarek-kalwa-space-base](https://user-images.githubusercontent.com/933396/81349655-6dcb0980-908e-11ea-83d1-7f150baafc30.jpg)

Reply:
We're way ahead of Elon!

Reply:
we really should make this `--space-1x` across the board.